integration/spark2/src/main/commonTo2.2And2.3/org/apache/spark/sql/hive/CarbonInMemorySessionState.scala

```diff
@@ -157,7 +158,13 @@ class InMemorySessionCatalog(
   }
 
   override def lookupRelation(name: TableIdentifier): LogicalPlan = {
-    val rtnRelation = super.lookupRelation(name)
+    val rtnRelation: LogicalPlan =
+      try {
+        super.lookupRelation(name)
+      } catch {
+        case ex: Exception =>
+          throw ex
```

Comment:
but here you are returning same type of exception object right? or am i missing anything?what about wraping this exception object with custom exception object ? or else i cannot see anything in this statement you are doing different